data/json/items/tool/metalworking.json

```diff
@@ -348,5 +348,20 @@
     "material": [ "steel" ],
     "symbol": "]",
     "color": "light_cyan"
+  },
+  {
+    "id": "casting_mold",
+    "type": "TOOL",
+    "name": { "str": "sand casting mold" },
+    "description": "A mold for casting molten metals into different shapes",
+    "weight": "540 g",
+    "volume": "400 ml",
```

Comment:
Please add a longest side.   In theory any casting recipes that need a piece cast longer than a couple inches shorter than this would need a larger casting mold, possibly going all the way up to furniture sized casting molds?

Reply:
You mean increase the size?  yes, that makes a lot of sense

Reply:
You can keep this one the size you have it at but add a longest side like this `"longest_side": "40 cm",`   Otherwise the game assumes whatever you've created is a cube that fits neatly into all kinds of containers that don't necessarily make sense.